src/engine/controls/bpmcontrol.cpp

```diff
@@ -381,6 +382,8 @@ double BpmControl::shortestPercentageChange(const double& current_percentage,
 }
 
 double BpmControl::calcSyncedRate(double userTweak) {
+    if (BPM_DEBUG) qDebug() << getGroup() << "BpmControl::calcSyncedRate, tweak " << userTweak;
+    m_dUserRateTweak = userTweak;
```

Comment:
This is only used as boolean to store the tweaked state during the engine callback. 
Maybe we should rename it like that. How about:  
bool m_userTweakingSync

Reply:
good eye!